pioneer/packages/joy-tokenomics/src/Landing/GetStarted.tsx

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import styled from 'styled-components';
+import usePromise from '@polkadot/joy-utils/react/hooks/usePromise';
+import { useTransport } from '@polkadot/joy-utils/react/hooks';
+
+const Subtitle = styled('h3')`
+    font-weight: 600;
+    color: rgba(0,0,0, 0.65);
+    margin-bottom: 1rem;
+`;
+
+const GetStarted = () => {
+  const transport = useTransport();
+  const [data] = usePromise(() => transport.proposals.proposalsBatch('Active'), undefined, []);
+
+  return <>
+    <p>If you&apos;re new to Joystream, go to our <a href='https://github.com/Joystream/helpdesk/'>Helpdesk</a> to learn what you can do, and how to get started. If you&apos;re not there already, join our <a href='https://t.me/joinchat/CNyeUxHD9H56m3e_44hXIA'>telegram group</a>, get some tokens, and join our community!</p>
+    <Subtitle>Membership</Subtitle>
+    <p>Most actions on the platform requires you to become a <a href='/#/members/edit'>Member</a>.</p>
+    <Subtitle>Council</Subtitle>
+    <p>As &quot;The video platfrom DAO&quot;, Joystreams day to day operations are controlled by the <a href='/#/council/members'>Council</a> through a Proposal system.</p>
+    <Subtitle>Proposals</Subtitle>
+    <p>There are currently {data?.proposals.length || 'no'} <a href='/#/proposals'>Proposals</a> open for voting.</p>
```

Comment:
Small thing here but would be nice to have the numbers in words (e.g. one two three four five) rather than figures.

Reply:
Since the data here is dynamic and not manually inserted by someone, we would need to write a function which turns numbers into words. After some research doesn't look like it would be even remotely simple. 

One saving grace here would be if there is already such a function in the codebase. Maybe @Lezek123 knows?